aredis_om/model/model.py

```diff
@@ -1156,7 +1203,7 @@ def Field(
         vector_options=vector_options,
         **current_schema_extra,
     )
-    field_info._validate()
+    # field_info._validate()
```

Comment:
Should this fire? Or are you doing validate elsewhere? Otherwise, no need for comment probably.